File: DataCenterWebApp/ClientApp/src/app/interfaces/systemoverview.ts
import { ISystemOverview } from './isystemoverview';

export class SystemOverview implements ISystemOverview {
  LastImportDate: Date;

  ExperimentCount: number;

  HighestSequenceID: number;

  ICDataCenterAddress: string;

  ICDataCenterVersion: string;

  ICDataCenterStatus: string;

  DataCenterWebAppAddress: string;

  DataCenterWebAppVersion: string;

  DataCenterWebAppStatus: string;

  LoggedOnUser: string;

  LoggedOnRole: string;

  LastUpdate: Date;

}

File: DataCenterLib/SystemOverview.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DataCenterLib
{
    public class SystemOverview
    {
        public string LastImportDate { get; set; }
        public int ExperimentCount { get; set; }
        public int HighestSequenceID { get; set; }
        public string ICDataCenterAddress { get; set; }
        public string ICDataCenterVersion { get; set; }
        public string ICDataCenterStatus { get; set; }
        public string DataCenterWebAppAddress { get; set; }
        public string DataCenterWebAppVersion { get; set; }
        public string DataCenterWebAppStatus { get; set; }
        public string LastUpdate { get; set; }
    }
}
